Horse.js

```diff
@@ -0,0 +1,5 @@
+function Horse() {
```

Comment:
I think you change the function name is myHorse().
I want you check the function name ?